src/supermarket/app/controllers/api/v1/cookbook_versions_controller.rb

```diff
@@ -1,7 +1,7 @@
 class Api::V1::CookbookVersionsController < Api::V1Controller
-  before_action :check_cookbook_name_present, only: [:foodcritic_evaluation, :collaborators_evaluation]
-  before_action :check_authorization, only: [:foodcritic_evaluation, :collaborators_evaluation]
-  before_action :find_cookbook_version, only: [:foodcritic_evaluation, :collaborators_evaluation]
+  before_action :check_cookbook_name_present, only: [:foodcritic_evaluation, :collaborators_evaluation, :publish_evaluation]
+  before_action :check_authorization, only: [:foodcritic_evaluation, :collaborators_evaluation, :publish_evaluation]
+  before_action :find_cookbook_version, only: [:foodcritic_evaluation, :collaborators_evaluation, :publish_evaluation]
```

Comment:
We could, but that would make it accessible outside of the controller.  This feels like something that should only be used in this controller, so I am leaning toward a private method.  Not sure if there is such a thing as a private constant.